src/contrib/cluster/Akka.Cluster.Sharding.Tests/InactiveEntityPassivationSpec.cs

```diff
@@ -132,7 +132,7 @@ public void Passivation_of_inactive_entities_must_passivate_entities_when_they_h
 
             // Make sure "1" hasn't seen a message in 3 seconds and passivates
             var timeSinceOneSawAMessage = DateTime.Now.Ticks - timeOneSawMessage;
-            probe.ExpectNoMsg(TimeSpan.FromSeconds(3) - TimeSpan.FromTicks(timeSinceOneSawAMessage));
+            probe.ExpectNoMsg(TimeSpan.FromSeconds(3) - TimeSpan.FromTicks(timeSinceOneSawAMessage) - TimeSpan.FromMilliseconds(40));
```

Comment:
The fix here was simple - this spec, when it'd fail on the build server, would be for reasons like this:

```
Failed: Expected no messages during 00:00:00.9951057, instead we received <1 passivating> from [akka://AkkaSpec/system/sharding/myType/1/1#392767797] after 00:00:00.9946593
Expected: True
Actual: False
```
The issue in this case is that the time we were setting during the `ExpectNoMsg` period is more precise than what's allowed by the server's clock resolution. Thus, I padded the time by an additional 40ms - which should be plenty enough to make sure the test is still meaningful but a big enough margin to avoid having the test too tightly fitted.